docs/tutorial/dark-mode.md

```diff
@@ -0,0 +1,183 @@
+# Dark Mode
+
+## Overview
+
+## macOS
+
+In macOS 10.14 Mojave, Apple introduced a new [system-wide dark mode][system-wide-dark-mode]
+for all macOS computers. If your Electron app has a dark mode, you can make it
+follow the system-wide dark mode setting using
+[the `nativeTheme` api](../api/native-theme.md).
+
+In macOS 10.15 Catalina, Apple introduced a new "automatic" dark mode option
+for all macOS computers. In order for the `nativeTheme.shouldUseDarkColors` and
+`Tray` APIs to work correctly in this mode on Catalina, you need to either have
+`NSRequiresAquaSystemAppearance` set to `false` in your `Info.plist` file, or
+use Electron `>=7.0.0`. Both [Electron Packager][electron-packager] and
```

Comment:
```suggestion
for all macOS computers. In order for the `nativeTheme.shouldUseDarkColors` and
`Tray` APIs to work correctly in this mode on Catalina, you need to use Electron
`>=7.0.0`, or set `NSRequiresAquaSystemAppearance` to `false`
in your `Info.plist` file for older versions. Both [Electron Packager][electron-packager] and
```

Suggestion because we should probably promote upgrading past Electron 6 at this point.